ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-release-1.1.yaml

```diff
@@ -0,0 +1,152 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+  kube-rbac-proxy:
+    name: "4.12"
+    namespace: ocp
+    tag: kube-rbac-proxy
+  ocp_builder_rhel-8-golang-1.19-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.12
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.openshift
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.12:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12
+  to: external-dns-operator
+operator:
+  bundles:
+  - as: external-dns-operator-bundle
+    base_index: operator-index
+    dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/openshift/origin-external-dns-operator:latest
+    with: pipeline:external-dns-operator
+  - pullspec: quay.io/openshift/origin-kube-rbac-proxy:latest
+    with: pipeline:kube-rbac-proxy
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    export GOLANGCI_LINT_CACHE=$(mktemp -d)
+    make verify
+  container:
+    from: src
+- as: unit
+  commands: |
+    export XDG_CACHE_HOME=$(mktemp -d)
+    make test
+  container:
+    from: src
+- as: e2e-aws-operator
```

Comment:
Is external-dns supported on ovn?

Reply:
You mean the network plugin? OVN is now the default network plugin of OpenShift (couldn't find from which version though). So, I suppose the answer is yes. Also because I cannot remember anything specific to a particular network plugin we used in the operator or controller.

Reply:
You should add the "ovn" to the test name where needed to make sure this runs the OVN tests without flakes. See https://github.com/openshift/release/pull/31430

Reply:
Done.